src/evaluation.py

```diff
@@ -76,30 +78,59 @@ async def a_generate(self, prompt: str, schema: BaseModel) -> BaseModel:
     def get_model_name(self):
         return "Llama-3.1 8B"
 
-rag_llm = RAG_LLM()
-eval_llm = EvalHuggingFaceLLM()
 
-questions = ["Wie viele Unterlagen des Finanzausschusses sind vorhanden und welche sind das?"]
-scores = []
-
-metric_faithful = FaithfulnessMetric(model=eval_llm)
+def evaluate_model(config: str, secrets: str) -> None:
+    
+    eval_llm = OllamaModel( # EvalHuggingFaceLLM()
+        model="llama3.2",
+        base_url="http://localhost:11434"
+    )
+    questions = ["Wie viele Unterlagen des Finanzausschusses sind vorhanden und welche sind das?"]
```

Comment:
Maybe it would be a good approach to have a list of of dicts with questions and answers that could be used for evaluation.
Could be an extra file evaluation.toml